app-emulation/fuse-utils/fuse-utils-1.3.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
+
+DESCRIPTION="Utils for the Free Unix Spectrum Emulator by Philip Kendall"
+HOMEPAGE="http://fuse-emulator.sourceforge.net"
+SRC_URI="mirror://sourceforge/fuse-emulator/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="audiofile gcrypt jpeg png zlib"
+
+RDEPEND="~app-emulation/libspectrum-1.3.5[gcrypt?]
```

Comment:
This would have to be updated every time upstream releases a new version of libspectrum (or at least when that happens and 1.3.5 has to be masked/removed, due to security issues for instance). Please change this to >= unless there are strong indications future versions of libspectrum are likely to break the API.